sensor_format.md

```diff
@@ -0,0 +1,163 @@
+The sensor data coming over the I2C connection includes all of the information to measure the amount of power the individual circuits are comsuming. It consists of a short header, power and current information for all nineteen sensors, and AC voltage/frequency information.
+
+## Overview
+
+The information is encoded little-endian, and is a total of 284 bytes. The diagram below is a to-scale overview of a message received from the sensor.
+
+```
+       +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
```

Comment:
Used the [specification for IPv6](https://datatracker.ietf.org/doc/html/rfc2460) for inspiration on how to visualize the format